release-notes/v1.5.6.yaml

```diff
@@ -0,0 +1,30 @@
+date: December 4, 2025
+
+# Changes that are expected to cause an incompatibility with previous versions, such as deletions or modifications to existing APIs.
+breaking changes: |
+
+# Updates addressing vulnerabilities, security flaws, or compliance requirements.
+security updates: |
+  Patches for EnvoyProxy CVE-2025-64527, CVE-2025-66220, and CVE-2025-64763. Ref https://github.com/envoyproxy/envoy/releases/tag/v1.35.7
+
+# New features or capabilities added in this release.
+new features: |
+
+bug fixes: |
+  - Fixed xDS snapshot cache to clear snapshots when streams close, preventing proxies from receiving stale configuration after reconnection (#6618)
+  - Fixed OIDC authentication endpoint being overwritten by discovered value (#7460)
+  - Fixed findOwningGateway to return controller based on linked GatewayClass (#7611)
+  - Fixed default namespace handling when namespace is unset (#7612)
+  - Fixed skeleton route status entries being created for unmanaged GatewayClasses (#7536)
+  - Fixed gateway status to treat too many addresses as programmed (#7542)
```

Comment:
Release-notes weren't added for all of the cherry-picked commits so I generated these. Please let me know if there's a more preferable way to handle this.

Reply:
Nit:
Should we remove "-"? It's for markdown.
I think issue id can be helpful, but all the previous release notes didn't include them.

Reply:
I like the fact that PR is linked to go to the source of the fix